README.md

```diff
@@ -1606,8 +1680,8 @@ A bit on how all this fits together:
     on income over $200,000 (single) or $250,000 (married filing jointly)
   - **3.8%**
     [Net Investment Income Tax](https://www.irs.gov/uac/Newsroom/Net-Investment-Income-Tax-FAQs)
-    on investment income if you make over $200,000 (single) or $250,000 (married filing
-    jointly)
+    (sometimes called the “Obamacare tax”) on investment income if you make over $200,000
```

Comment:
Remove reference to the "Obamacare tax" slang. People use that term to disparage healthcare taxes, and we don't want to editorialize.

Reply:
Reviewer suggestion. I do think it's worth linking the two terminologies, whatever your political position, as that term is used in the media and many people don't know what "net investment income tax" is or that it was linked to healthcare.